spec/manifest_spec.rb

```diff
@@ -226,6 +226,22 @@
       ])
     end
 
+    context 'when products is a string that is not a location' do
+      before { manifest_hash[:location] = 'cool_location' }
+
+      it 'returns no products without raising an error' do
+        expect(manifest.products).to eq([])
+      end
+    end
+
+    context 'when products is a string that matches a location' do
+      before { manifest_hash[:location] = 'ticket_sidebar' }
+
+      it 'defaults to support without raising an error' do
```

Comment:
Why would it default to support if it matches a valid location? It would detect the correct product right? The test doesn't change but the name of the test would

Reply:
Mmm. Unfortunately, I think it doesn't detect the correct product. If no product is specified ZAS assumes we're in support, then does a search for support locations which match the string in the manifest.